packages/codex-context/src/retriever.ts

```
@@ -44,7 +44,10 @@ export type Retriever = {
 
 export class SmartGptrRetriever implements Retriever {
   private fetcher: (url: string, init?: any) => Promise<any>;
-  private log = createLogger("codex-context", { component: "retriever" });
+  private log = createLogger({
+    service: "codex-context",
+    base: { component: "retriever" },
+  });
```

ESLint

[error] Property should have a readonly modifier.

(functional/prefer-immutable-types)